security-model.md

```diff
@@ -0,0 +1,105 @@
+# Security Model
+
+This security model aims to clarify the project's current security position against industry 
+recommendations and general security best practices. The information provided here should make 
+it easier for security teams from highly regulated sectors to assess and use SPO for their workloads.
+
+The SPO project strives to maintain the [least privilege principle], by trying to run with 
+the bare minimum privileges for the least amount of time possible. This approach decreases
+the attack surface and the window of opportunity in which it can be abused.
+
+Note that this is an on-going effort, and may change over time as new features are developed.
+
+## Host Isolation (a.k.a. Pod Security / Security Context)
+
+Details the security requirements to run SPO from a host isolation perspective. 
+
+### Initialisation
+
+Seccomp leverages an init container to setup an initial symlink as `root`, so it can then operate
+ as a non-root user.
+
+During initialisation it also uses the capabilities:
+
+- `CHOWN`
+- `FOWNER`
+- `FSETID`
+- `DAC_OVERRIDE`
+
+
+### Running Mode
+
+The running permissions for the three core technologies supported: 
+
+|                                  | Seccomp | SELinux | AppArmor |
+|----------------------------------|---------|---------|----------|
+|               Requires root user |    No   |   Yes   |    Yes   |
```

Comment:
The idea here was to be more generic, meaning that for SELinux feature to be operational, something must be running at that given permission level to be able support it.

Initially I thought as marking with `No*` and adding a caveat soon after, but then decided to make it easier. But I agree, it is not completely clear. 
Do you think it would be better if we broke down each of the components that support each enforcement technology (selinux, apparmor, selinux)?

Reply:
I think your approach is better and we might want to just add a side-note or a sub-paragraph later.